src/server_manager/electron_app/index.ts

```diff
@@ -175,6 +175,16 @@ function main() {
       electron.Menu.setApplicationMenu(electron.Menu.buildFromTemplate(menuTemplate));
     }
 
+    // Digital Ocean stopped sending 'Acces-Control-Allow-Origin' headers in some responses
```

Comment:
Can you move this to a function (something like `WorkaroundDigitalOceanCorsIssue()`) so it doesn't pollute the main()? It's also clearer what the full workaround code is.

Reply:
Done in 7bf510.